test/models/tasks/copy_media_task_test.rb

```diff
@@ -68,12 +70,15 @@
     episode.expect(:url, 'http://test-f.prxu.org/path/guid/audio.mp3')
 
     task.stub(:episode, episode) do
-      task.task_status_changed({}, 'complete')
+      task.update_attributes(status: 'complete')
+      # TODO: why do these fail?
+      # podcast.verify
+      # episode.verify
```

Comment:
Probably need to dig into this more.  But I'm not sure these tests were actually testing anything ... since they didn't call `verify` on the mock.  But when I do call verify, I get: 

```
MockExpectationError: expected publish!() => true
```

I can confirm publish is being called, so not sure what the problem is.  Is it trying to verify publish was called with `true`?